system/lib/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -427,7 +427,7 @@ struct __sanitizer_file_handle {
 };
 #endif
 
-#if SANITIZER_MAC
+#if SANITIZER_MAC || SANITIZER_EMSCRIPTEN
```

Comment:
Why did this change? I don't see anything obviously related to memory64 here. And the changes below move *away* from uptr, which is the opposite of what I'd expect...

Reply:
This was the solution discussed with @sbc100 elsewhere, which I can't seem to find right now in this forest of reviews. Essentially the changes needed for `MEMORY64` (to make `msg_iovlen` `unsigned` instead of `uptr`) are the same as those for the mac apparently, so we share this code.